src/plugin/ui/auth-menu.test.ts

```
@@ -1,10 +1,19 @@
 import { describe, it, expect } from 'vitest';
-import { ANSI } from './ansi';
+import type { AccountStatus, AccountInfo } from './auth-menu';
+
+// Duplicate private helpers for testing
+function sanitizeTerminalText(value: string | undefined): string | undefined {
+  if (!value) return undefined;
+  return value
+    .replace(/\x1b\[[0-?]*[ -/]*[@-~]/g, '')
```

Biome (2.4.4)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)